### PR TITLE
Fix linkcheck

### PR DESCRIPTION
**Pull request by stevepiercy**
This PR fixes broken or removes dead URLs in source code and docs. There is one weird one that I finally tracked down from Pyramid when building its docs in c6b6237.

### CODE REVIEW

File: src/webob/request.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def body_file_seekable(self):
         "int",
     )
     remote_user = environ_getter("REMOTE_USER", None)
-    remote_host = environ_getter("REMOTE_HOST", None, "4.1.9")
+    remote_host = environ_getter("REMOTE_HOST", None)
```
Comment (bertjwregeer) [inline src/webob/request.py]: This is already fixed in a different branch. Please remove this change for right now.

Comment (stevepiercy) [reply src/webob/request.py]: Done in d0eda05.

**Comment by bertjwregeer** [comment]
My bad on the Google Cloud build... I added that because I wanted to test it, but apparently it just fails the job if you don't have a `cloudbuild.yaml`

**Comment by stevepiercy** [comment]
Appveyor builds fail, too, but only on py2.7. I have no clue.

**Pull request merged**